rpm/src/main/jetty-config/resources/logging.properties

```diff
@@ -17,14 +17,15 @@ java.util.logging.FileHandler.formatter=java.util.logging.SimpleFormatter
 
 # Mute org.restlet.ext.httpclient.internal.HttpMethodCall sendRequest
 java.util.logging.FileHandler.filter=com.sixsq.slipstream.util.RestletSendRequestMuteFilter
+java.util.logging.FileHandler.filter=
```

Comment:
What is the rationale behind this change ?

Reply:
Good catch.  Cut & paste error.